### DISCUSSION TITLE
router.goForward()

### DISCUSSION THREAD

**Discussion by anonymouscatcher** [vercel/next.js#24299] [Help]
Hi There, 

I'm using Next.js to create an application, In my application I have back and forward arrows to go back and go forward exactly like browser, I have notice there's a method to return to previous pages but not for going forward. I have been using `react-router-dom` package before and there was a `goBack()` and `goForward()` method to handle this. 

Thanks in advance

**Comment by bobarros** [comment]
This method that goes back is for pagination or is just using browser history (the previous page visited)?

**Reply by anonymouscatcher** [reply]
browser history,

**Reply by bobarros** [reply]
Then you might try window.history.go(1) for forward and window.history.go(-1) to go back.
